inferlo/pairwise/optimization/convex_bounds_test.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+
+from inferlo import PairWiseFiniteModel
+from inferlo.pairwise.optimization.convex_bounds import LPrelaxation
+from inferlo.pairwise.testing import grid_potts_model, tree_potts_model, \
+    line_potts_model
+
+
+def test_grid_4x4x2():
+    model = grid_potts_model(4, 4, al_size=2, seed=0)
+    max_lh_gt = model.max_likelihood(algorithm='bruteforce')
+    max_lh_gt_value = np.log(model.evaluate(max_lh_gt))
+    max_lh_value_relax = LPrelaxation(model)
+    assert (max_lh_gt_value <= max_lh_value_relax)
```

Comment:
This looks wrong. On the right is scalar, on the left is 2d array.
It passes because result is np.array of boleans, and it is "truthy", but this dosn't verify correctness.

Instead, you want to evaluate energy at fractional point and assert that it's >= than max_lh_gt_value. You can't easily do it in test, so just add this value of energy (what prob.solve returns and add it here).